css-three-column-layout/styles.css

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+  font-family: Lato, sans-serif;
+  text-align: center;
+  color: white;
+  background-color: #009fe3;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 60px 20px;
+  width: 100%;
+}
+
+h1 {
+  font-size: 50px;
+  margin-bottom: 50px;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.column {
+  flex: 1;
+  flex-basis: 400px;
+  max-width: 400px;
+  margin: 30px;
+  padding: 30px;
+  background-color: #009fe3;
+  text-align: center;
+}
+
```

Comment:
One last thing I missed the first time, you should be more specific and set this as a layout class instead. Instead of `column` you have 3 columns so you should have this as `column-one-third` and set the `flex-basis` based on needed 3 column. (Just like how we did it in the `css-layout-classes`) This is so that the class is easily reusable if you need it in the future. You also don't need a `max-width` here and you could remove that, but everything that isn't `flex-basis` should be moved into a new class and you should just have the layout class with flex-basis.